installer/src/config/resources/jbdevstudio.bat

```diff
@@ -1 +0,0 @@
-start "Red Hat Developer Studio" "%~dp0\studio\jbdevstudio.exe"
```

Comment:
Yes, the file is renamed to devstudio.bat. Apparently github didn't notice it was a renamed file and is showing you a delete instead.